packages/tldraw/src/lib/ui/components/Toolbar/ToggleToolLockedButton.tsx

```diff
@@ -25,11 +28,16 @@ export function ToggleToolLockedButton({ activeToolId }: ToggleToolLockedButtonP
 
 	if (!activeToolId || !tool.isLockable) return null
 
+	const toggleLockAction = actions['toggle-tool-lock']
+	const tooltipContent = toggleLockAction?.kbd
+		? `${msg('action.toggle-tool-lock')} ${kbdStr(toggleLockAction.kbd)}`
+		: msg('action.toggle-tool-lock')
+
 	return (
-		<TldrawUiTooltip content={msg('action.toggle-tool-lock')}>
+		<TldrawUiTooltip content={tooltipContent}>
 			<TldrawUiButton
 				type="normal"
-				title={msg('action.toggle-tool-lock')}
+				title={tooltipContent}
```

Comment:
drive-by comment: let's nix this `title` - we forgot to remove it when we added tooltips, it's just redundant now!

Reply:
no worries done!